src/runtime/entries/cloudflare.ts

```diff
@@ -3,12 +3,21 @@ import {
   getAssetFromKV,
   mapRequestToAsset,
 } from "@cloudflare/kv-asset-handler";
+import { addEventListener } from "@cloudflare/workers-types";
 import { withoutBase } from "ufo";
 import { requestHasBody } from "../utils";
 import { nitroApp } from "#internal/nitro/app";
 import { useRuntimeConfig } from "#internal/nitro";
 import { getPublicAssetMeta } from "#internal/nitro/virtual/public-assets";
 
+addEventListener("scheduled", (event) => {
+  return nitroApp.hooks.callHook("cloudflare:scheduled", event);
```

Comment:
Can you make sure if the format is same or similar to workers?

Reply:
They have different arguments, but typing doesn't work properly. Should we go with `cloudflare-module:scheduled` and `cloudflare:scheduled` instead?

Reply:
> Is the event type also different between them?

According to Typescript, they are the same for scheduled. But they aren't for queues.

Reply:
At least object syntax would make them more similar... This is a low-level hook anyway btw so i won't worry too much about differences. We can set an indicator key like `isWorker`/`isModule` if needed.

Reply:
```ts
interface CloudflareModuleRuntimeHooks {
  "cloudflare:scheduled": (moduleArgs: {
    controller: ScheduledController;
    env: CloudflareModuleEnv;
    context: ExecutionContext;
  }) => any;
  "cloudflare:queue": (moduleArgs: {
    batch: MessageBatch;
    env: CloudflareModuleEnv;
    context: ExecutionContext;
  }) => any;
}

interface CloudflareSWRuntimeHooks {
  "cloudflare:scheduled": (event: ScheduledEvent) => any;
  "cloudflare:queue": (event: QueueEvent) => any;
}
```

This is their signature at the moment, they don't have any common properties. 

Note that `addEventEventListener` can accept a 3rd optional argument that can't be specified easily with hooks :

```ts
export interface EventTargetAddEventListenerOptions {
  capture?: boolean;
  passive?: boolean;
  once?: boolean;
  signal?: AbortSignal;
}
```